slack_bolt/version.py

```diff
@@ -1,3 +1,3 @@
 """Check the latest version at https://pypi.org/project/slack-bolt/"""
 
-__version__ = "1.25.0"
+__version__ = "1.26.0.dev4"
```

Comment:
suggestion: We'll need to revert this back to the original version 🙂

Reply:
@mwbrooks Thanks so much for catching this 🙏 ✨

Reply:
Woo! Thanks for the co-author!